Modules/ArcGIS/DSCResources/ArcGIS_Portal/ArcGIS_Portal.psm1

```diff
@@ -548,10 +548,9 @@ function Set-TargetResource {
             if (-not($sysProps)) {
                 $sysProps = @{ }
             }
-            $ExpectedWebContextUrl = "https://$($ExternalDNSName)/$($PortalContext)"
-            $ExpectedPrivatePortalUrl = "https://$($PortalEndPoint):7443/arcgis" # "https://$($ExternalDNSName)/$($PortalContext)" 
-            if (($sysProps.WebContextURL -ine $ExpectedWebContextUrl) -or ($sysProps.privatePortalURL -ine $ExpectedPrivatePortalUrl)) {
-				
+            
+            if ($ExternalDNSName){
+                $ExpectedWebContextUrl = "https://$($ExternalDNSName)/$($PortalContext)"
                 Write-Verbose "One of the system properties for WebContextURL '$($sysProps.WebContextURL)' or privatePortalURL '$($sysProps.privatePortalURL)' does not match expected values"
```

Comment:
Its not intuitive to me, why 
` if ($ExternalDNSName){ `
results in
`  Write-Verbose "One of the system properties for WebContextURL '$($sysProps.WebContextURL)' or privatePortalURL '$($sysProps.privatePortalURL)' does not match expected values"  `

Probably Debug message Needs to be adjusted.

Reply:
Yes this debug message was split, but I forgot to remove the message. Will remove this.